app/models/event.rb

```diff
@@ -22,7 +22,7 @@ class Event < ApplicationRecord
   default_scope       -> { where(label: Whitelabel[:label_id]) }
 
   scope :with_topics, -> { joins(:topics).distinct }
-  scope :current,     -> { where(date: Date.today.to_time..(Time.now + 9.weeks)).limit(1).order('date ASC') }
+  scope :current,     -> { where('date >= ?', Date.today.to_time).limit(1).order('date ASC') }
```

Comment:
```suggestion
  scope :current,     -> { where(date: Date.today.to_time...).limit(1).order(date: :asc) }
```

Reply:
haha now @salzig joins the conversation on `where(date: Date.today..)` vs `where('date > ?', Date.today)` :laughing: 

Talking about the ordering:  I'd do a separate commit that updates this everywhere. How do you feel about dropping the `:asc` part as this is the default behavior of AR?

Reply:
I prefer to skip it, but I know that a lot of people prefer `:asc` to be present.